vizro-core/examples/_chart-gallery/utils/chart_factory.py

```diff
@@ -0,0 +1,359 @@
+    return vm.Page(
+        id=id,
+        title=title,
+        layout=vm.Layout(grid=PAGE_GRID),
+        components=[
+            vm.Card(
+                text="""
+
+            #### What is a column chart?
+
+            A column chart is a vertical bar chart, with column lengths varying according to the
+            categorical value they represent. The scale is presented on the y-axis, starting with zero.
+
+            &nbsp;
+
+            #### When to use it?
+
+            Select a column chart when you want to help your audience draw size comparisons and identify
+            patterns between categorical data, i.e., data that presents **how many?** in each category. You can
+            arrange your columns in any order to fit the message you wish to emphasize. Be mindful of
+            labeling clearly when you have a large number of columns. You may need to include a legend,
+            or use abbreviations in the chart with fuller descriptions below of the terms used.
+        """
+            ),
+            vm.Graph(
+                figure=px.bar(
+                    data_frame=DATA_DICT["tips_agg"],
+                    y="total_bill",
+                    x="day",
+                )
+            ),
+            CodeClipboard(
+                text="""
+                ```python
+                import vizro.models as vm
+                import vizro.plotly.express as px
+                from vizro import Vizro
+
+                tips = px.data.tips()
+                tips_agg = (
+                    tips.groupby("day")
+                    .agg({"total_bill": "sum"})
+                    .sort_values("total_bill")
+                    .reset_index()
+                )
+
+                page = vm.Page(
+                    title="Column",
+                    components=[
+                        vm.Graph(
+                            figure=px.bar(
+                                data_frame=tips_agg,
+                                y="total_bill",
+                                x="day",
+                            )
+                        )
+                    ],
+                )
+
+                dashboard = vm.Dashboard(pages=[page])
+                Vizro().build(dashboard).run()
+                ```
+                """
+            ),
+        ],
+    )
+
+
+def treemap_factory(id: str, title: str):
+    """Reusable function to create the page content for the treemap chart with a unique ID."""
+    return vm.Page(
+        id=id,
+        title=title,
+        layout=vm.Layout(grid=PAGE_GRID),
+        components=[
+            vm.Card(
+                text="""
+
+                #### What is a treemap?
+
+                A treemap shows hierarchical data arranged as a set of nested rectangles: rectangles sized
+                proportionately to the quantity they represent, combining together to form larger **parent**
+                category rectangles.
```

Comment:
A treemap shows hierarchical data arranged as a set of nested rectangles: rectangles are sized proportionately to the quantity they represent, combined together to form larger **parent** category rectangles. - my suggestion, but can be omitted.